PodcastGenerator/core/episodes.php

```diff
@@ -92,6 +92,106 @@ function getEpisodes($category = null, $_config)
     return $episodes_data;
 }
 
+function searchEpisodes($name = "", $_config)
+{
+    $name = strtolower($name);
+    $supported_extensions = simplexml_load_file($_config['absoluteurl'] . 'components/supported_media/supported_media.xml');
+    $realsupported_extensions = array();
+    foreach ($supported_extensions as $item) {
+        array_push($realsupported_extensions, $item->extension);
+    }
+    $supported_extensions = $realsupported_extensions;
+    unset($realsupported_extensions);
+
+    // Get episodes names and pubDates (which are the file
+    // modification times).  We'll ignore files with future
+    // timestamps.
```

Comment:
Do we want to ignore future timestamps? This might be something useful

Reply:
This a good question. Currently future timestamps are being handles like if the file would not exist, however changing this is the scope of another PR

Reply:
Signed in admin accounts will now see future timestamped episodes as well from [commit aed82f0](https://github.com/PodcastGenerator/PodcastGenerator/pull/290/commits/aed82f0171a2db37dd3eda8967f0e312957f432c) and [commit 1931cc2](https://github.com/PodcastGenerator/PodcastGenerator/pull/290/commits/1931cc2fdfa836d22c6560aa9873f6dee45ba689).